<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionffff"/>
    <addaction name="actionInfo_about_figure"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCircle"/>
    <addaction name="actionEightStar"/>
    <addaction name="actionFiveStar"/>
    <addaction name="actionHexagon"/>
    <addaction name="actionParallelogram"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionRomb"/>
    <addaction name="actionSixStar"/>
    <addaction name="actionSquare"/>
    <addaction name="actionTriangle"/>
    <addaction name="actionMy_figure"/>
   </widget>
   <widget class="QMenu" name="menuMove">
    <property name="title">
     <string>Move</string>
    </property>
    <addaction name="actionUp"/>
    <addaction name="actionDown"/>
    <addaction name="actionRight"/>
    <addaction name="actionLeft"/>
   </widget>
   <widget class="QMenu" name="menuSize">
    <property name="title">
     <string>Size</string>
    </property>
    <addaction name="actionIncrease"/>
    <addaction name="actionDecrease"/>
   </widget>
   <widget class="QMenu" name="menuRotation">
    <property name="title">
     <string>Rotation</string>
    </property>
    <addaction name="actionRight_turn"/>
    <addaction name="actionLeft_turn"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuMove"/>
   <addaction name="menuSize"/>
   <addaction name="menuRotation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionffff">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionInfo_about_figure">
   <property name="text">
    <string>Info about figure</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionEightStar">
   <property name="text">
    <string>EightStar</string>
   </property>
  </action>
  <action name="actionFiveStar">
   <property name="text">
    <string>FiveStar</string>
   </property>
  </action>
  <action name="actionHexagon">
   <property name="text">
    <string>Hexagon</string>
   </property>
  </action>
  <action name="actionParallelogram">
   <property name="text">
    <string>Parallelogram</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionRomb">
   <property name="text">
    <string>Romb</string>
   </property>
  </action>
  <action name="actionSixStar">
   <property name="text">
    <string>SixStar</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="text">
    <string>Square</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
  <action name="actionMy_figure">
   <property name="text">
    <string>My figure</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="text">
    <string>Up</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="text">
    <string>Down</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionIncrease">
   <property name="text">
    <string>Increase</string>
   </property>
  </action>
  <action name="actionDecrease">
   <property name="text">
    <string>Decrease</string>
   </property>
  </action>
  <action name="actionRight_turn">
   <property name="text">
    <string>Right turn</string>
   </property>
  </action>
  <action name="actionLeft_turn">
   <property name="text">
    <string>Left turn</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionffff</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCleanClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInfo_about_figure</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onInfoClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCircle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCircleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEightStar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onEightStarClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMy_figure</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMyFigureClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTriangle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onTriangleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSquare</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSquareClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSixStar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSixStarClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRomb</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRombClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRectangle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRectangleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionParallelogram</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onParallelogramClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHexagon</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onHexagonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFiveStar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFiveStarClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDecrease</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDecreaseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDown</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDownClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionIncrease</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onIncreaseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLeft</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onLeftClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLeft_turn</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onLeftTurnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRight</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRightClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRight_turn</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRightTurnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onUpClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCleanClicked()</slot>
  <slot>onInfoClicked()</slot>
  <slot>onCircleClicked()</slot>
  <slot>onEightStarClicked()</slot>
  <slot>onFiveStarClicked()</slot>
  <slot>onHexagonClicked()</slot>
  <slot>onParallelogramClicked()</slot>
  <slot>onRectangleClicked()</slot>
  <slot>onRombClicked()</slot>
  <slot>onSixStarClicked()</slot>
  <slot>onSquareClicked()</slot>
  <slot>onTriangleClicked()</slot>
  <slot>onMyFigureClicked()</slot>
  <slot>onUpClicked()</slot>
  <slot>onDownClicked()</slot>
  <slot>onLeftClicked()</slot>
  <slot>onRightClicked()</slot>
  <slot>onIncreaseClicked()</slot>
  <slot>onDecreaseClicked()</slot>
  <slot>onRightTurnClicked()</slot>
  <slot>onLeftTurnClicked()</slot>
 </slots>
</ui>
